frontend/src/firebase/firebaseInteractor.ts

```diff
@@ -100,7 +100,9 @@ export default class FirebaseInteractor {
     userAuth.user.sendEmailVerification();
     if (accountType === "STUDENT") {
       let initialAssessmentId = await this.createInitialAssessment();
+      let currentDaysActive = new Array<String>();
```

Comment:
Can we use the array literal syntax here?
```suggestion
      let currentDaysActive = [];
```